sklearn/preprocessing/tests/test_data.py

```diff
@@ -701,6 +703,62 @@ def test_scaler_without_centering():
     assert_array_almost_equal(X_csc_scaled_back.toarray(), X)
 
 
+def _check_attributes_scalers(scaler_1, scaler_2):
```

Comment:
nitpick: This sounds French to me. `_check_scalers_attributes` is more natural I think.

Reply:
Also if the function is only valid for the identity case, we should probably reflect that in the function name:

`_check_identity_scalers_attributes`